README.md

```diff
@@ -1,3 +1,12 @@
+***User Support:***
+
+
+For user questions please look for answers and ask first in the GATK forum: https://gatk.broadinstitute.org/hc/en-us
```

Comment:
Should this link directly to the forum instead of the splash page?  https://gatk.broadinstitute.org/hc/en-us/community/topics

Also, do we want to have the link destination visible or just a link on GATK forum?  That seems more inline with the rest of the readme.
```suggestion
For user questions please look for answers and ask first in the [GATK forum](https://gatk.broadinstitute.org/hc/en-us).
```